Scripts/DCS-BIOS/lib/modules/aircraft_modules/M-2000C.lua

```diff
@@ -149,6 +153,45 @@ local function build_pcn_segments(pcn, pcn_segment, display_name, length, includ
 	end
 end
 
+local function display_matrix_to_string(displayMatrix)
+	-- Mapping from binary segment pattern to character
+	local segment_to_char = {
```

Comment:
It's probably worth pulling this out of this function so we don't have to reallocate it every frame.